### DISCUSSION TITLE
Connecting the ESP8266 power supply directly from Epever (RJ45) Is it safe?

### DISCUSSION THREAD

**Discussion by arek1194** [softwarecrash/EPEver2MQTT#38] [Q&A]
In the cabling documentation for your project, you present the possibility of connecting the ES8266 power supply from the Epever RJ45 plug. It would be very convenient, but while browsing similar projects, I came across a warning - do not connect the power supply from RJ45, because it only has a current capacity of 50mA (7.5V). Therefore, I would like to make sure if it is safe?
It also provides a link with such a warning
https://github.com/Bettapro/Solar-Tracer-Blynk-V3/blob/v3.x.x/docs/Esp8266_max485_epever_rj45.md

Apart from that, very good job. Regards
![epeveru 2024-03-14 130910](https://github.com/softwarecrash/EPEver2MQTT/assets/44841235/23c11237-9944-42af-a657-016530346dfe)

**Comment by hedley-a** [comment]
I have used the XTRA 1206 Supply, 5 Volts and if I remember correctly ~100mA  - I tested using  an active load.
Do though, add a 220uF decoupling capacitor close to the ESP module.

Point three - the author is  referring to  ground reference not ground loops an entirely different thing. RS422 being balanced has a high degree of noise immunity. You MUST take care that any other ESP inputs or outputs are referenced to the same ground otherwise magic  smoke is released :-(  use opto-isolators.

The  ST3485 RS422-TTL converter works well and is actually specified for 3.0 Volts unlike the modules which use 5 Volt IC's.

**Comment by thitcher** [comment]
For me it works without problems for months now

**Comment by softwarecrash** [comment]
> In the cabling documentation for your project, you present the possibility of connecting the ES8266 power supply from the Epever RJ45 plug. It would be very convenient, but while browsing similar projects, I came across a warning - do not connect the power supply from RJ45, because it only has a current capacity of 50mA (7.5V). Therefore, I would like to make sure if it is safe? It also provides a link with such a warning https://github.com/Bettapro/Solar-Tracer-Blynk-V3/blob/v3.x.x/docs/Esp8266_max485_epever_rj45.md
> 
> Apart from that, very good job. Regards ![epeveru 2024-03-14 130910](https://private-user-images.githubusercontent.com/44841235/312832008-23c11237-9944-42af-a657-016530346dfe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA0Mjk4NzQsIm5iZiI6MTcxMDQyOTU3NCwicGF0aCI6Ii80NDg0MTIzNS8zMTI4MzIwMDgtMjNjMTEyMzctOTk0NC00MmFmLWE2NTctMDE2NTMwMzQ2ZGZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE0VDE1MTkzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhM2M5MDUyY2M4NzdlMDQ0ODFiNjUyZTA0ZjMwODJkZTY5Y2QxM2JhZGM3NGFiMzhiNTNmYTM0NmIxMjRlMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.k1jVf_743bttUPfXL4X-yi0rIFvwy1Awa9XPY-H7t5o)

its tricky.... some epever deliver not enugh for the esp, deepens on the used esp. other epevers delivery a lot more than 50mah, so just test it. or use external power source
